tests/scanner/test_data/fake_groups_scanner_data.py

```diff
@@ -36,144 +37,87 @@
   member: bbbbb@mycompany.com
 """
 
+class FakeGroup:
+    def __init__(self, name, member_name, type, direct_member_count):
+        self.name = name
+        self.member_name = member_name
+        self.type = type
+        self.direct_member_count = direct_member_count
+
+
 ALL_GROUPS = (
-    {'direct_member_count': 3L,
-     'group_email': 'aaaaa@mycompany.com',
-     'group_id': 'aaaaa',
-     'group_kind': 'admin#directory#group',
-     'id': 1L,
-    },
-    {'direct_member_count': 3L,
-     'group_email': 'bbbbb@mycompany.com',
-     'group_id': 'bbbbb',
-     'group_kind': 'admin#directory#group',
-     'id': 2L,
-    },
-    {'direct_member_count': 3L,
-     'group_email': 'ccccc@mycompany.com',
-     'group_id': 'ccccc',
-     'group_kind': 'admin#directory#group',
-     'id': 3L,
-    },
-    {'direct_member_count': 3L,
-     'group_email': 'ddddd@mycompany.com',
-     'group_id': 'ddddd',
-     'group_kind': 'admin#directory#group',
-     'id': 4L,
-    },
+    FakeGroup('aaaaa', 'aaaaa@mycompany.com', 'admin#directory#group', 3L),
+    FakeGroup('bbbbb', 'bbbbb@mycompany.com', 'admin#directory#group', 3L),
```

Comment:
The direct_member_count doesn't seem right compare with the data below, BBB and DDD have more direct_member_count than 3.

Reply:
This is not used, so removing.  The exact members will be checked with a direct comparison.